File: Codechef/ATM/ATM.java
//TODO: Print balance after ATM transaction

import java.util.*;

class Codechef {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		//TODO: Getting user input
		int amount = sc.nextInt();
		double initial_balance = sc.nextDouble();
		
		//TODO: Checking and printing balance after transaction
		if (amount % 5 == 0 && initial_balance >= ((double) amount + 0.50)) {
			double balance = initial_balance - ((double) amount + 0.50);
			System.out.printf("%.2f", balance);
		} else {
			System.out.println(initial_balance);
		}
	}
}

File: Codechef/ATM/ATM.c
//TODO: Print balance after transaction

#include <stdio.h>

int main(void) 
{

    int a;
    float b;
    scanf("%d %f",&a,&b);                //TODO: Getting user input
    if(a%5==0 && a+0.5<=b)
        printf("%.2f",b-a-0.5);          //TODO: Printing balance
    else
        printf("%.2f",b);

	return 0;
}


